utils/sane-backends/Makefile

```diff
@@ -130,6 +130,20 @@ $(call Package/sane-backends/Default)
   USERID=saned:saned
 endef
 
+define Package/sane-daemon-avahi-dbus-service
+$(call Package/sane-backends/Default)
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS+=sane-daemon +avahi-dbus-daemon
+  TITLE+= (network daemon avahi service files)
+endef
+
+define Package/sane-daemon-avahi-nodbus-service
```

Comment:
I did some tests on PROVIDES:

``
root@OpenWrt:~# opkg install avahi-daemon
Installing avahi-dbus-daemon...
``

So, if no variation defined, it is free to choose any (in this case, dbus variation).

After I select one of them (nodbus variant), avahi-daemon is already provided

``
root@OpenWrt:~# opkg install avahi-nodbus-daemon
(...)
root@OpenWrt:~# opkg install avahi-daemon
Package avahi-nodbus-daemon is already installed on root.
``

I still can ask to install both nodbus and dbus variants, but it will fail afterwards because they have conflicting files.

Now using it with DEPENDS:

I installed nut-avahi-service, which depends on avahi-daemon:

With no avahi installed:

opkg install nut-avahi-service
Installing nut-avahi-service...
Installing avahi-nodbus-daemon...
Installing libavahi-nodbus-support...

Now, I installed the dbus variant and the nut-avahi-service after:

``
opkg install avahi-dbus-daemon nut-avahi-service
Installing avahi-dbus-daemon
Installing libavahi-dbus-support
Installing nut-avahi-service
``

IMHO, it just works as expected. Why the same thing that works for openssh, nut, mpd does not work for sane-backends?